File: commands/tick.js
export const tickLocalTask = (taskName, json, HOME) => {
    for (let i = 0; i < json.tasks.length; i++) {
        const task = json.tasks[i];
        
        if (task.name === taskName) {
            task.finished = true
        }

        const data = JSON.stringify(json)

        // WRITE
        Deno.writeTextFileSync(`${HOME}/tascli.json`, data)
    }
}

File: lib/commands/tick.dart
import 'package:tascli/helpers/json.dart';

void tickLocalTask(json, taskName) {
  for (int i = 0; i < json['tasks'].length; i++) {
    if (json['tasks'][i]['name'] == taskName) {
      json['tasks'][i]['finished'] = true;
    }
  }

  writeJson(json);
}